infra/resources/_modules/monitoring/variables.tf

```
@@ -0,0 +1,9 @@
+variable "resource_group_name" {
+  type        = string
+  description = "Resource group to deploy resources to"
+}
+
+variable "alert_iocom_error_notification_slack_email" {
```

In my opinion we should not tie a variable to its specific use. In the future the slack e-mail can be referenced in other places!

```suggestion
variable "iocom_slack_email" {
```